models/agent_work_time/int_zendesk__ticket_work_time_business.sql

```diff
@@ -39,11 +39,11 @@ with ticket_historical_status as (
 
     select 
       ticket_status_crossed_with_schedule.*,
-      ({{ fivetran_utils.timestamp_diff(
-              "" ~ dbt_utils.date_trunc('week', 'ticket_status_crossed_with_schedule.status_schedule_start') ~ "", 
-              'ticket_status_crossed_with_schedule.status_schedule_start',
-              'second') }} /60
-            ) as start_time_in_minutes_from_week,
+    ({{ fivetran_utils.timestamp_diff(
+            "cast(" ~ dbt_date.week_start('ticket_status_crossed_with_schedule.status_schedule_start','UTC') ~ "as " ~ dbt_utils.type_timestamp() ~ ")", 
```

Comment:
Both fields needed to be cast as timestamp or else the calculations would be off.

Reply:
@fivetran-jamie exactly!